client/k8s/application/Application.go

```diff
@@ -54,10 +54,11 @@ type K8sRequestBean struct {
 }
 
 type PodLogsRequest struct {
-	SinceTime     *metav1.Time `json:"sinceTime,omitempty"`
-	TailLines     int          `json:"tailLines"`
-	Follow        bool         `json:"follow"`
-	ContainerName string       `json:"containerName"`
+	SinceTime         *metav1.Time `json:"sinceTime,omitempty"`
+	TailLines         int          `json:"tailLines"`
+	Follow            bool         `json:"follow"`
+	ContainerName     string       `json:"containerName"`
+	PrevContainerLogs bool         `json:"previous"`
```

Comment:
By looking at variable name it does not strike as boolean value, please name it appropriately. maybe isPrevContainerLogsEnabled or any other name like this.

Reply:
done